src/extension.ts

```diff
@@ -17,12 +17,13 @@ export async function activate(ctx: ExtensionContext) {
   const onUpdate = async () => {
     CORE = await init();
     DISPOSABLES.forEach(i => i.dispose());
-    DISPOSABLES = [...registerCompletions(ctx, CORE), ...registerCommands(ctx, CORE)];
+    DISPOSABLES.length = 0;
```

Comment:
is this the same as
```
DISPOSABLES = [];
```

Reply:
Nope, it creates a new instance and lost the reference and that's actually the reason of this bug